libr/bin/format/elf/elf.c

```diff
@@ -1014,6 +1014,11 @@ ut64 Elf_(r_bin_elf_get_baddr)(struct Elf_(r_bin_elf_obj_t) *bin) {
 			}
 		}
 	}
+	if (base == UT64_MAX  && bin->ehdr.e_type == ET_REL) {
+		//we return our own base address for ET_REL type
+		//we act as a loader for ELF
+		return 0x08000000;
```

Comment:
Noticed that everything is now relocated to 0x8000000.
What is the benefit of doing something like this? Wouldn't it make it even harder (well, to those who aren't familiar with this) to share information between standard tools and r2?